### PR TITLE
Make powershell the default profile

### PR DESCRIPTION
**Pull request by matthew4850**
Sets powershell as the default profile.
#431

### CODE REVIEW

File: src/cascadia/TerminalApp/CascadiaSettings.cpp [hunk 1]
```diff
@@ -121,14 +121,12 @@ void CascadiaSettings::_CreateDefaultSchemes()
 void CascadiaSettings::_CreateDefaultProfiles()
 {
     Profile defaultProfile{};
```
Comment (gpotter2) [inline src/cascadia/TerminalApp/CascadiaSettings.cpp]:
```suggestion
    Profile cmdProfile{};
```

**Comment by gpotter2** [comment]
For a weird reason, the diff on github doesn't show any spacing changes, but it actually happens anyway

**Comment by HBelusca** [comment]
I see that "cmd.exe" is hardcoded in quite some places; isn't it possible to use whatever is specified in the %COMSPEC% environment variable instead?

File: src/cascadia/TerminalApp/CascadiaSettings.cpp [hunk 2]
```diff
@@ -120,15 +120,13 @@ void CascadiaSettings::_CreateDefaultSchemes()
...
-    Profile defaultProfile{};
-    defaultProfile.SetFontFace(L"Consolas");
-    defaultProfile.SetCommandline(L"cmd.exe");
-    defaultProfile.SetColorScheme({ L"Campbell" });
-    defaultProfile.SetAcrylicOpacity(0.75);
-    defaultProfile.SetUseAcrylic(true);
-    defaultProfile.SetName(L"cmd");
-
-    _globals.SetDefaultProfile(defaultProfile.GetGuid());
+     Profile cmdProfile{};
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/CascadiaSettings.cpp]: hyper-nit: the indenting of this line looks like it might be wrong

Comment (gpotter2) [inline src/cascadia/TerminalApp/CascadiaSettings.cpp]:
```suggestion
    Profile cmdProfile{};
```

Comment (matthew4850) [reply src/cascadia/TerminalApp/CascadiaSettings.cpp]: This has now been fixed.

File: src/cascadia/TerminalApp/CascadiaSettings.cpp [hunk 3]
```diff
@@ -150,8 +148,10 @@ void CascadiaSettings::_CreateDefaultProfiles()
     powershellProfile.SetUseAcrylic(false);
     powershellProfile.SetName(L"PowerShell");
 
-    _profiles.emplace_back(defaultProfile);
+    _profiles.emplace_back(cmdProfile);
```
Comment (miniksa) [inline src/cascadia/TerminalApp/CascadiaSettings.cpp]: Do these need to be inserted in the other order? That is, will this leave cmd.exe as the profile on ctrl+shift+1?

If PowerShell is the default, it probably belongs on Ctrl+Shift+1 by default as well (by nature of the ordering).

Comment (matthew4850) [reply src/cascadia/TerminalApp/CascadiaSettings.cpp]: Thanks for pointing out, I have now updated this.

**Pull request merged**